Psychtoolbox/PsychDemos/ProceduralSmoothedDiscMaskDemo.m

```diff
@@ -0,0 +1,107 @@
+function ProceduralSmoothedDiscMaskDemo(ndiscs)
+% ProceduralSmoothedDiscMaskDemo([ndiscs=700]) -- An aquarium full of discs 
+% within a centered mask!
+%
+% This demo shows how to use the Screen('DrawTextures') command to draw a
+% large number of similar images quickly - in this case, smooth edged
+% discs masked by another smoothed disc.
+
+% History:
+% 25/04/2018 created (Junxiang Luo)
+
+% Setup defaults and unit color range:
+PsychDefaultSetup(2);
+
+% Disable synctests for this quick demo:
+oldSyncLevel = Screen('Preference', 'SkipSyncTests', 2);

```

Comment:
synctests setting not restored at the end or in case of error.

Reply:
Fixed